### PR TITLE
Fill readme.md small refactor, make pytest work from console

### PR DESCRIPTION
**Pull request by jereml99**

### CODE REVIEW

**Review by FreddieHorn — DISMISSED** [review]
We can use word "microfilter" referring to these modular functions like char_swap, remove_spaces etc. The reason is that it sounds better marketing-wise. But it is up to discussion I guess.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+```
+
+Konsolowy interfejs pozwala na wybranie jednego z 2 źródeł wejścia programu:
+- plik `wejscie.txt` znajdujący się w głównym folderze projektu
+- Wejście użytkownika z konsoli
+
+Program wypisuje na wyjściu konsoli tekst z przekleństwami zamienionymi na gwiazdki (*)
+. 
+
+Lista obsługiwanych przez nas wulgaryzmów dostępna jest w pliku `data/badwords.yaml`
```
Comment (SunBear1) [inline README.md:26]: Maybe lets enumerate these words inside the readme file?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+
+###Testy
+W celu przedstawienia możliwości filtra przygotowaliśmy testy jednostkowe. Aby je uruchomić należy wywołać komendę:
```
Comment (SunBear1) [inline README.md:29]: These e2e tests aren't unit tests btw

**Review by SunBear1 — COMMENTED** [review]
All good, a little nitpicks from my side

**Comment by SunBear1** [comment]
CONFLICTS. WHY THEY ARE NOT RESOLVED? DO IT. NOW.

**Review by SunBear1 — APPROVED** [review]

**Pull request merged**